lib/c-api/Cargo.toml

```diff
@@ -102,6 +102,7 @@ wasmer-artifact-create = ["wasmer-compiler/wasmer-artifact-create"]
 static-artifact-load = ["wasmer-compiler/static-artifact-load"]
 static-artifact-create = ["wasmer-compiler/static-artifact-create"]
 webc_runner = ["virtual-fs", "webc"]
+dylib = []
```

Comment:
This feature was deprecated long ago. We shouldn't add it back

Reply:
You use it in https://github.com/wasmerio/wasmer/blob/d431db9a3cbcf9a34ce3f5e83c5deddac29b52f7/lib/c-api/src/wasm_c_api/engine.rs#L258 - generating compiler warning because `dylib` is not declared. Should it be removed there?